matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/model/EventRedactBody.kt

```diff
@@ -25,5 +25,10 @@ internal data class EventRedactBody(
         val reason: String? = null,
 
         @Json(name = "org.matrix.msc3912.with_relations")
+        val unstableWithRelations: List<String>? = null,
+
+        @Json(name = "with_rel_types")
         val withRelations: List<String>? = null,
-)
+) {
+    fun getBestWithRelations() = withRelations ?: unstableWithRelations
```

Comment:
This function should select the right option according to the stability of the feature